### DISCUSSION TITLE
Sort APIs alphabetically in API dropdown list

### DISCUSSION THREAD

**Discussion by Ledger94** [Azure/api-management-developer-portal#2609] [Feature request]
Feature Request:

Sort APIs in API drop down list alphabetically or have an option to enable this.

**Comment by ghost** [comment]
@Ledger94, thank you for opening this issue. We will triage it within the next few business days.

**Comment by harunrst** [comment]
Thanks, @Ledger94. We will take a look into this.

**Comment by ghost** [comment]
@Ledger94, thank you for requesting this feature.

**Comment by senpudev-hobbie** [comment]
Hello! I was going to try to add this feature but apparently you already have it implemented.

![image](https://github.com/Azure/api-management-developer-portal/assets/109302954/56414172-96b8-4c82-bf91-f3e4c1b7c470)
